src/xunit.performance.analysis/Program.cs

```diff
@@ -116,21 +120,25 @@ private static int Main(string[] args)
 
             var comparisonResults = DoComparisons(allComparisonIds, testResults);
 
+        #if !LINUX_BUILD
```

Comment:
how about using the #if contruct only to stub out the WeiteXXX methods?  so we have less #if overall...